docx/header.py

```diff
@@ -8,12 +8,11 @@
     absolute_import, division, print_function, unicode_literals
```

Comment:
@scanny ok i think this commit is good to go. unittest and acceptance test both pass.

I went with ElementProxy instead of object for Header's base class just so that I could pass sectPr without writing some custom init method.

Class methods should be in alphabetical order. I didn't worry about tests though. It seems like those aren't alphabetical currently.

`lazyproperty` is cool! I have done that a bunch but I never thought to abstract it out like that. Super handy.

What's next? The end goal right now is "Able to load a document with header text and access that text" right?

Lots of steps to get there.